index.html

```diff
@@ -8,10 +8,26 @@
     <title>Awesome Books</title>
   </head>
   <body>
-    <div class="center">
-      <h1>All awesome books</h1>
-      <div class="container"></div>
-      <h2>Add new books</h2>
+    <header>
+      <nav>
+          <div class="container-nav-menu">
+              <div class="logo">Awesome Books</div>  
+              <ul class="menu__items">
+                  <li class="menu__items-item">List</li>
+                  <li class="menu__items-item">Add New</li>
+                  <li class="menu__items-item">Contact</li>
+              </ul>
+          </div>
+      </nav>
+  </header>
+  <div class="list-list">
+  <h1 class="head">All awesome books</h1>
+  <div class="container">
+  </div>
+  </div>
+
+    <div class="add-form">
+      <h2 class="head">Add new books</h2>
       <form class="form">
         <input
           class="title"
```

Comment:
- [ ] Great job so far @Team. You've done a great job so far. Your project looks great. But there are just a few changes you need to make to successfully complete this milestone :100: 
- [ ] First, of all, I will suggest that you center the form properly and ensure that it takes up as much space as seen on the page in the given wireframe :100: 
- [ ] From the images below, you can also see that the nav links on the right of the page are missing the vertical bars between the nav links so kindly include this as well :+1: 
- [ ] Lastly, from the given wireframe, on each of the pages, there needs to be a footer. But you're currently missing this so I will suggest you include this in your design as well :100: 

| Your Design | The Wireframe |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/234584624-94da0368-8900-4625-be66-893860a58f23.png) | ![image](https://user-images.githubusercontent.com/102699958/234584779-8ad7085a-6fc1-45de-a357-e5643b997213.png) |

Reply:
well resolved! ✔️